module/VuFind/src/VuFind/Autocomplete/Solr.php

```diff
@@ -171,21 +178,51 @@ protected function initSearchObject()
     /**
      * Process the user query to make it suitable for a Solr query.
      *
-     * @param string $query Incoming user query
+     * @param string $query  Incoming user query
+     * @param ?array $extras Array of extra parameter
      *
-     * @return string       Processed query
+     * @return string        Processed query
      */
-    protected function mungeQuery($query)
+    protected function mungeQuery(string $query, array $extras = null): string
```

Comment:
Would it make sense to default $extras to `[]` instead of `null` so you don't have to use the `?array` type above? Also, I wonder if `$options` would be a better name than `$extras`.